pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+requires = ["setuptools>=65", "wheel>=0.38"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "neural-lam"
+version = "0.1.0"
+description = "Neural Weather Prediction for Limited Area Modeling"
+readme = "README.md"
+keywords = [
+    "numericalweatherpredictions",
+    "machinelearning",
+    "deeplearning",
+    "weather",
+]
+classifiers = [
+    "Development Status :: Alpha",
+    "Intended Audience :: Developers",
+    "Natural Language :: English",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+]
+requires-python = ">=3.10"
+authors = [{ name = "Joel Oskarsson", email = "joel.oskarsson@liu.se" }]
+
+[project.urls]
+source = "https://github.com/joeloskarsson/neural-lam"
+documentation = "https://github.io/MeteoSwiss-APN/neural-lam"
+
+[tool.black]
+line-length = 100
+
+[tool.isort]
+default_section = "THIRDPARTY"
+profile = "black"
+# Headings
+import_heading_stdlib = "Standard library"
+import_heading_thirdparty = "Third-party"
+import_heading_firstparty = "First-party"
+import_heading_localfolder = "Local"
+# Known modules to avoid misclassification
+known_standard_library = [
+    # Add standard library modules that may be misclassified by isort
+]
+known_third_party = [
+    # Add third-party modules that may be misclassified by isort
+]
+known_first_party = [
+    # Add first-party modules that may be misclassified by isort
+    "neural_lam",
+]
+
+
+[tool.flake8]
+max-line-length = 100
```

Comment:
Any specific reason for 100 here? I had tried to stick to 90 before (due to convenience with my workspace setup), but if we now enforce this (which I think is good), would it make more sense to use 80 as prescribed by PEP8?

Reply:
no reason! please use 80